### ISSUE TITLE
Support for .NET Core 3

### ISSUE THREAD

**Issue by seanmars** [thomaslevesque/NHotkey#16]
Can supported for .net standard? Thanks

**Comment by thomaslevesque** [comment]
Hi @seanmars,

What do you need exactly? What kind of app are you building?
I'm not sure a .NET Standard version would make sense. NHotkey only works on Windows, in Windows Forms or WPF apps. Until recently, it was only possible to write such applications with the .NET Framework. Now you can also use .NET Core 3, so I guess it would make sense to make a netcoreapp3.0 version. Would that work for you?

In the meantime, if you're targeting netcoreapp3.0, you should be able to use the current package, because .NET Core 2+ can reference .NET Framework assemblies.

**Comment by seanmars** [comment]
Hi @thomaslevesque ,

I'm using WPF with .NET core 3(netcoreapp3.1); It's worked fine. But still get this warning:

```log
Severity	Code	Description	Project	File	Line	Suppression State
Warning	NU1701	Package 'NHotkey.Wpf 1.2.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETCoreApp,Version=v3.1'. This package may not be fully compatible with your project.	WpfApp1	C:\WpfApp1\WpfApp1\WpfApp1.csproj	1	
```

My .csproj file is below:

```xml
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="NHotkey.Wpf" Version="1.2.1" />
  </ItemGroup>

</Project>
```

**Comment by thomaslevesque** [comment]
Hi @seanmars,

I think you can safely ignore this warning. The library isn't using any exotic API, so it should work fine on .NET Core 3. I'll eventually add explicit support for .NET Core 3, but I won't be able to do it before at least a month.

**Comment by thomaslevesque** [comment]
This is fixed in [2.0.0-beta.1](https://github.com/thomaslevesque/NHotkey/releases/tag/2.0.0-beta.1)
